vendor/bat-native-ads/src/bat/ads/internal/features/conversions/conversions_features.cc

```diff
@@ -21,8 +21,7 @@ const int kDefaultResourceVersion = 1;
 const char kFieldTrialParameterDefaultConversionIdPattern[] =
     "conversions_default_conversion_id_pattern";
 const char kDefaultDefaultConversionIdPattern[] =
-    "<meta.*name=\"ad-conversion-id\".*content=\"(.*)\".*>";
-
+    "<meta.*name=\"ad-conversion-id\".*content=\"([^\"]*)\".*>";
```

Comment:
I just remember that we also had an issue matching against HTML using single quotes, i.e. `<meta name='ad-conversion-id' ... >`, but I don't think we need to fix as we can tell advertisers to include with double quotes. Wdyt?